[0m17:36:56.788128 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D3F4430B30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D3F4981070>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D3F21957F0>]}


============================== 17:36:56.811673 | 66621673-b151-449b-a98e-c1daa6507b17 ==============================
[0m17:36:56.811673 [info ] [MainThread]: Running with dbt=1.8.7
[0m17:36:56.814189 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt init tn_job_portal', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m17:36:57.176014 [debug] [MainThread]: Starter project path: C:\Users\varsh\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\dbt\include\starter_project
[0m17:36:57.224006 [info ] [MainThread]: 
Your new dbt project "tn_job_portal" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:36:57.224006 [info ] [MainThread]: Setting up your profile.
[0m17:41:32.644608 [info ] [MainThread]: Profile tn_job_portal written to C:\Users\varsh\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m17:41:32.651944 [debug] [MainThread]: Command `dbt init` succeeded at 17:41:32.651944 after 276.09 seconds
[0m17:41:32.655990 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D3F046CE90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D3FE857230>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D3F2D5F2C0>]}
[0m17:41:32.655990 [debug] [MainThread]: Flushing usage events
[0m17:41:40.371207 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B5DE010380>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B5DA6610A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B5DCEA14C0>]}


============================== 17:41:40.378857 | 34fa728b-6356-406f-b029-fe3bb67ab783 ==============================
[0m17:41:40.378857 [info ] [MainThread]: Running with dbt=1.8.7
[0m17:41:40.378857 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m17:41:40.419821 [info ] [MainThread]: dbt version: 1.8.7
[0m17:41:40.419821 [info ] [MainThread]: python version: 3.12.10
[0m17:41:40.421830 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m17:41:40.421830 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m17:41:41.124454 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m17:41:41.124454 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m17:41:41.129985 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m17:41:41.130720 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m17:41:41.131989 [debug] [MainThread]: Command `dbt debug` failed at 17:41:41.131989 after 1.03 seconds
[0m17:41:41.133109 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B5DE2059A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B5FCAB89E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002B5DCDD7CB0>]}
[0m17:41:41.133109 [debug] [MainThread]: Flushing usage events
[0m22:40:10.706123 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CF0002A3F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CF005F8320>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CF007744A0>]}


============================== 22:40:10.706123 | 5a4298b1-b90b-4523-8a4c-994a67a71ef7 ==============================
[0m22:40:10.706123 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:40:10.706123 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:40:10.751697 [info ] [MainThread]: dbt version: 1.8.7
[0m22:40:10.751697 [info ] [MainThread]: python version: 3.12.10
[0m22:40:10.751697 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m22:40:10.751697 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m22:40:12.346937 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m22:40:12.350786 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m22:40:12.350786 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m22:40:12.350786 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:40:12.350786 [debug] [MainThread]: Command `dbt debug` failed at 22:40:12.350786 after 1.83 seconds
[0m22:40:12.350786 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CF00628980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CF1FEB80B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CF21090680>]}
[0m22:40:12.350786 [debug] [MainThread]: Flushing usage events
[0m22:42:34.972458 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002DF8D8056A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002DF924A75C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002DF924A6390>]}


============================== 22:42:34.972458 | aaeb71a5-bcd8-43ef-838c-4713c541c0e0 ==============================
[0m22:42:34.972458 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:42:34.979435 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m22:42:34.979435 [info ] [MainThread]: dbt version: 1.8.7
[0m22:42:34.996274 [info ] [MainThread]: python version: 3.12.10
[0m22:42:34.996274 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m22:42:34.998281 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m22:42:35.682616 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m22:42:35.684621 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m22:42:35.684621 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m22:42:35.684621 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:42:35.684621 [debug] [MainThread]: Command `dbt debug` failed at 22:42:35.684621 after 0.87 seconds
[0m22:42:35.684621 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002DF920FD0A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002DFB12E0380>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002DF924A7C50>]}
[0m22:42:35.684621 [debug] [MainThread]: Flushing usage events
[0m22:44:05.249979 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D66752D700>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D663DA9C40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D667A43980>]}


============================== 22:44:05.257891 | 4bac0207-6581-4619-b642-33063b301510 ==============================
[0m22:44:05.257891 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:44:05.257891 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m22:44:05.271301 [info ] [MainThread]: dbt version: 1.8.7
[0m22:44:05.272517 [info ] [MainThread]: python version: 3.12.10
[0m22:44:05.272517 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m22:44:05.272517 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m22:44:05.923561 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m22:44:05.923561 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m22:44:05.923561 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m22:44:05.923561 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:44:05.923561 [debug] [MainThread]: Command `dbt debug` failed at 22:44:05.923561 after 0.82 seconds
[0m22:44:05.923561 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D66752D700>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D663415C70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D681AC37D0>]}
[0m22:44:05.923561 [debug] [MainThread]: Flushing usage events
[0m22:45:17.277469 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BCDD1057F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BCE0BC7740>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BCE0BC54C0>]}


============================== 22:45:17.288893 | 02410bac-683c-4e9d-a0b7-f121d2165fd6 ==============================
[0m22:45:17.288893 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:45:17.288893 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:45:17.307395 [info ] [MainThread]: dbt version: 1.8.7
[0m22:45:17.309817 [info ] [MainThread]: python version: 3.12.10
[0m22:45:17.312268 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m22:45:17.315038 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m22:45:18.065284 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m22:45:18.065284 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m22:45:18.065284 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m22:45:18.065284 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:45:18.065284 [debug] [MainThread]: Command `dbt debug` failed at 22:45:18.065284 after 0.95 seconds
[0m22:45:18.065284 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BCE24289B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BCFEDB8C20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BD0104E7B0>]}
[0m22:45:18.065284 [debug] [MainThread]: Flushing usage events
[0m22:46:48.214623 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000169237CEFC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016925D06DE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016925E1CFB0>]}


============================== 22:46:48.218677 | 8991c23f-a57e-4b24-964d-681f80ffce62 ==============================
[0m22:46:48.218677 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:46:48.218677 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:46:48.224501 [info ] [MainThread]: dbt version: 1.8.7
[0m22:46:48.224501 [info ] [MainThread]: python version: 3.12.10
[0m22:46:48.231323 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m22:46:48.231323 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m22:46:48.901024 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m22:46:48.901024 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m22:46:48.901024 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m22:46:48.901024 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:46:48.901024 [debug] [MainThread]: Command `dbt debug` failed at 22:46:48.901024 after 0.87 seconds
[0m22:46:48.901024 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016928AA3740>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016920CB5C70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016949897CB0>]}
[0m22:46:48.910716 [debug] [MainThread]: Flushing usage events
[0m22:47:38.172239 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161E3F2CA10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161E3F2F560>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161E5805700>]}


============================== 22:47:38.172239 | af98c2dc-2cea-4699-a48c-01a5eeef56c9 ==============================
[0m22:47:38.172239 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:47:38.172239 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt debug --profiles-dir C:\\Users\\varsh\\.dbt', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:47:38.181524 [info ] [MainThread]: dbt version: 1.8.7
[0m22:47:38.181524 [info ] [MainThread]: python version: 3.12.10
[0m22:47:38.195565 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m22:47:38.196823 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m22:47:38.838790 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m22:47:38.838790 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m22:47:38.838790 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m22:47:38.838790 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m22:47:38.838790 [debug] [MainThread]: Command `dbt debug` failed at 22:47:38.838790 after 0.86 seconds
[0m22:47:38.838790 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161E5804F80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161E581B770>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000161864F61E0>]}
[0m22:47:38.838790 [debug] [MainThread]: Flushing usage events
[0m22:51:35.445267 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C1ABC86B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C1F1D5EE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C1F0D71D0>]}


============================== 22:51:35.454199 | da9f80ad-a6b2-4fa9-b161-8f30b77eda0f ==============================
[0m22:51:35.454199 [info ] [MainThread]: Running with dbt=1.8.7
[0m22:51:35.454199 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt debug --profiles-dir C:\\Users\\varsh\\.dbt', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:51:35.473070 [info ] [MainThread]: dbt version: 1.8.7
[0m22:51:35.476329 [info ] [MainThread]: python version: 3.12.10
[0m22:51:35.477494 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m22:51:35.478563 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m22:51:36.191878 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m22:51:36.191878 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m22:51:36.191878 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m22:51:36.200785 [info ] [MainThread]: adapter type: snowflake
[0m22:51:36.200785 [info ] [MainThread]: adapter version: 1.8.4
[0m22:51:36.204406 [info ] [MainThread]: Configuration:
[0m22:51:36.206156 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m22:51:36.206156 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m22:51:36.206156 [info ] [MainThread]: Required dependencies:
[0m22:51:36.208166 [debug] [MainThread]: Executing "git --help"
[0m22:51:36.267803 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m22:51:36.267803 [debug] [MainThread]: STDERR: "b''"
[0m22:51:36.267803 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m22:51:36.271493 [info ] [MainThread]: Connection:
[0m22:51:36.272081 [info ] [MainThread]:   account: SRACHMF-VO77249
[0m22:51:36.272081 [info ] [MainThread]:   user: VARSHINIPALURI
[0m22:51:36.272081 [info ] [MainThread]:   database: tn_job_portal
[0m22:51:36.272081 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m22:51:36.272081 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m22:51:36.272081 [info ] [MainThread]:   schema: stage
[0m22:51:36.272081 [info ] [MainThread]:   authenticator: None
[0m22:51:36.272081 [info ] [MainThread]:   oauth_client_id: None
[0m22:51:36.277825 [info ] [MainThread]:   query_tag: None
[0m22:51:36.278097 [info ] [MainThread]:   client_session_keep_alive: False
[0m22:51:36.278097 [info ] [MainThread]:   host: None
[0m22:51:36.278097 [info ] [MainThread]:   port: None
[0m22:51:36.278097 [info ] [MainThread]:   proxy_host: None
[0m22:51:36.279886 [info ] [MainThread]:   proxy_port: None
[0m22:51:36.279886 [info ] [MainThread]:   protocol: None
[0m22:51:36.279886 [info ] [MainThread]:   connect_retries: 1
[0m22:51:36.279886 [info ] [MainThread]:   connect_timeout: None
[0m22:51:36.279886 [info ] [MainThread]:   retry_on_database_errors: False
[0m22:51:36.282498 [info ] [MainThread]:   retry_all: False
[0m22:51:36.282498 [info ] [MainThread]:   insecure_mode: False
[0m22:51:36.282498 [info ] [MainThread]:   reuse_connections: None
[0m22:51:36.282498 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m22:51:36.801476 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m22:51:36.801476 [debug] [MainThread]: Using snowflake connection "debug"
[0m22:51:36.801476 [debug] [MainThread]: On debug: select 1 as id
[0m22:51:36.801476 [debug] [MainThread]: Opening a new connection, currently in state init
[0m22:51:39.198164 [debug] [MainThread]: SQL status: SUCCESS 1 in 2.391 seconds
[0m22:51:39.200266 [debug] [MainThread]: On debug: Close
[0m22:51:39.364731 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m22:51:39.364731 [info ] [MainThread]: [31m1 check failed:[0m
[0m22:51:39.370493 [info ] [MainThread]: Project loading failed for the following reason:
 project path <D:\DE\dataingestion\dbt_project.yml> not found

[0m22:51:39.373149 [debug] [MainThread]: Command `dbt debug` failed at 22:51:39.373149 after 4.11 seconds
[0m22:51:39.374392 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m22:51:39.374392 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C1ABC86B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C3EE644A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023C3ED545C0>]}
[0m22:51:39.374392 [debug] [MainThread]: Flushing usage events
[0m23:00:29.149755 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D29551160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D2BCEE930>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D2BCEE660>]}


============================== 23:00:29.161570 | 194d49ad-b54b-4ed8-927b-e2a4527d6bc3 ==============================
[0m23:00:29.161570 [info ] [MainThread]: Running with dbt=1.8.7
[0m23:00:29.161570 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt seed', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m23:00:29.164244 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\DE\dataingestion\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m23:00:29.164244 [debug] [MainThread]: Command `dbt seed` failed at 23:00:29.164244 after 0.15 seconds
[0m23:00:29.164244 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D29551160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D2B585DC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D2D1273E0>]}
[0m23:00:29.164244 [debug] [MainThread]: Flushing usage events
[0m23:01:03.589518 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E22F3FD820>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E22CDE0D40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E22CF301A0>]}


============================== 23:01:03.589518 | 0dced819-2e56-4b2f-ad3a-8dfbb9a1347f ==============================
[0m23:01:03.589518 [info ] [MainThread]: Running with dbt=1.8.7
[0m23:01:03.589518 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt seed', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m23:01:03.589518 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\DE\dataingestion\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m23:01:03.589518 [debug] [MainThread]: Command `dbt seed` failed at 23:01:03.589518 after 0.14 seconds
[0m23:01:03.589518 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E22F82C9B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E22CB3D280>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E22F6EBE60>]}
[0m23:01:03.589518 [debug] [MainThread]: Flushing usage events
[0m23:05:19.636561 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028558A3EF30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028558886DE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002855A041A30>]}


============================== 23:05:19.644493 | 0267d7d0-2bb0-4600-88fb-37394474afa3 ==============================
[0m23:05:19.644493 [info ] [MainThread]: Running with dbt=1.8.7
[0m23:05:19.644493 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m23:05:19.666029 [info ] [MainThread]: dbt version: 1.8.7
[0m23:05:19.667128 [info ] [MainThread]: python version: 3.12.10
[0m23:05:19.667665 [info ] [MainThread]: python path: C:\Users\varsh\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe
[0m23:05:19.668618 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m23:05:20.321877 [info ] [MainThread]: Using profiles dir at C:\Users\varsh\.dbt
[0m23:05:20.321877 [info ] [MainThread]: Using profiles.yml file at C:\Users\varsh\.dbt\profiles.yml
[0m23:05:20.321877 [info ] [MainThread]: Using dbt_project.yml file at D:\DE\dataingestion\dbt_project.yml
[0m23:05:20.321877 [info ] [MainThread]: adapter type: snowflake
[0m23:05:20.321877 [info ] [MainThread]: adapter version: 1.8.4
[0m23:05:20.321877 [info ] [MainThread]: Configuration:
[0m23:05:20.321877 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m23:05:20.321877 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m23:05:20.329206 [info ] [MainThread]: Required dependencies:
[0m23:05:20.329206 [debug] [MainThread]: Executing "git --help"
[0m23:05:20.380615 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m23:05:20.380615 [debug] [MainThread]: STDERR: "b''"
[0m23:05:20.380615 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m23:05:20.380615 [info ] [MainThread]: Connection:
[0m23:05:20.380615 [info ] [MainThread]:   account: SRACHMF-VO77249
[0m23:05:20.380615 [info ] [MainThread]:   user: VARSHINIPALURI
[0m23:05:20.380615 [info ] [MainThread]:   database: tn_job_portal
[0m23:05:20.391193 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m23:05:20.391193 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m23:05:20.392628 [info ] [MainThread]:   schema: raw
[0m23:05:20.392628 [info ] [MainThread]:   authenticator: None
[0m23:05:20.392628 [info ] [MainThread]:   oauth_client_id: None
[0m23:05:20.392628 [info ] [MainThread]:   query_tag: None
[0m23:05:20.395489 [info ] [MainThread]:   client_session_keep_alive: False
[0m23:05:20.395489 [info ] [MainThread]:   host: None
[0m23:05:20.395489 [info ] [MainThread]:   port: None
[0m23:05:20.395489 [info ] [MainThread]:   proxy_host: None
[0m23:05:20.395489 [info ] [MainThread]:   proxy_port: None
[0m23:05:20.395489 [info ] [MainThread]:   protocol: None
[0m23:05:20.395489 [info ] [MainThread]:   connect_retries: 1
[0m23:05:20.395489 [info ] [MainThread]:   connect_timeout: None
[0m23:05:20.399333 [info ] [MainThread]:   retry_on_database_errors: False
[0m23:05:20.399745 [info ] [MainThread]:   retry_all: False
[0m23:05:20.400335 [info ] [MainThread]:   insecure_mode: False
[0m23:05:20.400335 [info ] [MainThread]:   reuse_connections: None
[0m23:05:20.400335 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m23:05:20.711636 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m23:05:20.711636 [debug] [MainThread]: Using snowflake connection "debug"
[0m23:05:20.713644 [debug] [MainThread]: On debug: select 1 as id
[0m23:05:20.713644 [debug] [MainThread]: Opening a new connection, currently in state init
[0m23:05:21.386748 [debug] [MainThread]: SQL status: SUCCESS 1 in 0.673 seconds
[0m23:05:21.386748 [debug] [MainThread]: On debug: Close
[0m23:05:21.504372 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m23:05:21.504999 [info ] [MainThread]: [31m1 check failed:[0m
[0m23:05:21.505615 [info ] [MainThread]: Project loading failed for the following reason:
 project path <D:\DE\dataingestion\dbt_project.yml> not found

[0m23:05:21.505615 [debug] [MainThread]: Command `dbt debug` failed at 23:05:21.505615 after 2.22 seconds
[0m23:05:21.505615 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m23:05:21.508198 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028558A3E420>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028558A3EF30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028558A3FB60>]}
[0m23:05:21.508198 [debug] [MainThread]: Flushing usage events
[0m15:30:56.178323 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015A4D2B17F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015A48A49E80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015A4D9A0E30>]}


============================== 15:30:56.196754 | b89f2500-d3e8-4e51-807e-0f33949bc82d ==============================
[0m15:30:56.196754 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:30:56.197339 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt compile -m stg_job_postings_final', 'send_anonymous_usage_stats': 'True'}
[0m15:30:56.197339 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\DE\dataingestion\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:30:56.198606 [debug] [MainThread]: Command `dbt compile` failed at 15:30:56.198606 after 0.26 seconds
[0m15:30:56.198606 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015A4D28B8F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015A4A5008C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000015A48A49E80>]}
[0m15:30:56.198606 [debug] [MainThread]: Flushing usage events
[0m15:31:05.852003 [debug] [MainThread]: Error sending anonymous usage statistics. Disabling tracking for this execution. If you wish to permanently disable tracking, see: https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats.
[0m16:55:22.142601 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025A7A2077D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025A7A204C80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025A7A205190>]}


============================== 16:55:22.154503 | 3226235c-4cbe-462e-b6a1-7f687aa50cb9 ==============================
[0m16:55:22.154503 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:55:22.158350 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\varsh\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run -m stg_job_postings_final', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:55:22.158350 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\DE\dataingestion\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:55:22.158350 [debug] [MainThread]: Command `dbt run` failed at 16:55:22.158350 after 0.15 seconds
[0m16:55:22.158350 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025A780C0AD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025A78E96FF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025A76C1D520>]}
[0m16:55:22.158350 [debug] [MainThread]: Flushing usage events
[0m16:55:27.574938 [debug] [MainThread]: Error sending anonymous usage statistics. Disabling tracking for this execution. If you wish to permanently disable tracking, see: https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats.
